pyteal/ast/router.py

```diff
@@ -447,6 +450,7 @@ def __init__(
         self.approval_ast = ASTBuilder()
         self.clear_state_ast = ASTBuilder()
 
+        self.methods: list[ABIReturnSubroutine] = []
```

Comment:
nit: would be better to maintain a list of `sdk_abi.Method` than a list of `ABIReturnSubroutine`.